psalm.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<psalm xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="https://getpsalm.org/schema/config" errorLevel="7" resolveFromConfigFile="true" xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd">
+    <projectFiles>
+        <directory name="src"/>
+        <ignoreFiles>
+            <directory name="vendor"/>
+        </ignoreFiles>
+    </projectFiles>
+    <issueHandlers>
+        <UndefinedClass>
+            <errorLevel type="suppress">
+                <!-- NEXT_MAJOR: Remove next issue -->
+                <referencedClass name="Sonata\EasyExtendsBundle\Mapper\DoctrineCollector"/>
```

Comment:
I would have use the baseline for this for consistency...

Reply:
It is consistent with the reasons given in https://github.com/sonata-project/SonataTranslationBundle/pull/481#discussion_r613796784.

[Psalm docs says](https://psalm.dev/docs/running_psalm/dealing_with_code_issues/):

> There are two ways to suppress an issue – via the Psalm config or via a function docblock. 

In this case, specifying `<referencedClass name="Sonata\EasyExtendsBundle\Mapper\DoctrineCollector"/>` means that if this class is used somewhere else, there is no need of adding a new error.

I see the baseline as... let's increase `psalm` to level 3 for example, ignore all current errors so new code will be at level 3 and for the ignored code we will take a look when we can.

But if we are gradually increasing the level while fixing errors, I think the config file is the place.

Reply:
> In this case, specifying <referencedClass name="Sonata\EasyExtendsBundle\Mapper\DoctrineCollector"/> means that if this class is used somewhere else, there is no need of adding a new error.

This is useful indeed.

On the opposite, will the suppress in the issueHandler reported when it become unused (like the baseline does) ? 

Would you move some of the errors I put in the baseline https://github.com/sonata-project/SonataAdminBundle/blob/master/psalm-baseline.xml into the handler instead ?

Reply:
> On the opposite, will the suppress in the issueHandler reported when it become unused (like the baseline does) ?

AFAIK, psalm does not report these suppressed issues from baseline nor config, only from annotations in the code when the `findUnusedPsalmSuppress` option is `true`.

Reply:
Just tried and yes indeed. Maybe phpstan does, that's why I believed this.